pyproject.toml

```diff
@@ -6,32 +106,152 @@ use_parentheses = true
 ensure_newline_before_comments = true
 line_length = 88
 
-[tool.pylint.messages_control]
```

Comment:
Since starbase doesn't have pylint, please keep these pylint settings around.

Reply:
But we don't use pylint?

Reply:
pylint added